rabbitmq/kubedb_client_builder.go

```diff
@@ -114,13 +114,9 @@ func (o *KubeDBClientBuilder) GetRabbitMQClient() (*Client, error) {
 	authSecret := &core.Secret{}
 	var username, password string
 	if !o.db.Spec.DisableSecurity {
-		if o.db.Spec.AuthSecret == nil {
-			klog.Info("Auth-secret not set")
-			return nil, errors.New("auth-secret is not set")
-		}
 		err := o.kc.Get(o.ctx, types.NamespacedName{
 			Namespace: o.db.Namespace,
-			Name:      o.db.Spec.AuthSecret.Name,
+			Name:      o.db.GetAuthSecretName(),
```

Comment:
If user provide custom auth secret. will getAuthSecretName() function return the correct secret name?

Reply:
yes it will